From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Alex deVries <alex.devries@windriver.com>
Date: Sun, 9 Jun 2024 14:02:57 +0200
Subject: [PATCH] 6625: Docs: Add instruction for MacOS to avoid unverified
 developer error

Add instruction for when installing on MacOS to fix non-verified binaries
---
 GhidraDocs/InstallationGuide.html | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/GhidraDocs/InstallationGuide.html b/GhidraDocs/InstallationGuide.html
index 22bcd35509..448fa75509 100644
--- a/GhidraDocs/InstallationGuide.html
+++ b/GhidraDocs/InstallationGuide.html
@@ -112,6 +112,10 @@ Ghidra team if you have a specific need.</p></blockquote>
 <h2><a name="Install"></a>Installing Ghidra</h2>
 <p>To install Ghidra, simply extract the Ghidra distribution file to the desired filesystem
 destination using any unzip program (built-in OS utilities, 7-Zip, WinZip, WinRAR, etc)</p>
+
+<p>On MacOS, if you downloaded the distribution file with a browser, you should run
+<i>xattr -d com.apple.quarantine ghidra_&lt;version&gt;_&lt;date&gt;.zip</i> before unzipping.</p>
+
 <h3><a name="InstallationNotes"></a>Installation Notes</h3>
 <ul>
   <li>
-- 
2.45.1

